### ISSUE TITLE
Printing Wrong Loss

### ISSUE THREAD

**Issue by myavkat** [kazuto1011/svm-pytorch#4]
While printing the loss of the epoch in the train function, sum_loss is divided by N which is the dataset's size. But the losses are added to sum_loss N/args.batchsize times.
